system-tests/project-fixtures/svelte/src/Store.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+import { messageStore } from './store'
+
+let message;
```

Comment:
I am surprised this works, I wonder how Svelte knows how to trigger reactivity... I guess it assumes `let` variables trigger a re-render.

Reply:
Yeah seems `let` is where the magic happens: https://svelte.dev/tutorial/writable-stores